Sources/TokamakCore/Renderer.swift

```diff
@@ -66,4 +66,14 @@ public protocol Renderer: AnyObject {
     with host: MountedHost,
     completion: @escaping () -> ()
   )
+
+  /** Returns a body of a given pritimive view, or `nil` if `view` is not a primitive view for
+   this renderer.
+   */
+  func body(for view: Any) -> AnyView?
```

Comment:
Would it make sense for this to be called `primitiveBody(for:)`? Then it’s more clear that it only returns the body of a primitive view?